.github/workflows/claude-nl-suite.yml

```
@@ -70,28 +69,94 @@ jobs:
+            elif grep -qi '<Signature>' "$f"; then
+              echo "ok=true" >> "$GITHUB_OUTPUT"
+            else
+              echo "ok=false" >> "$GITHUB_OUTPUT"
+            fi
+
+        # --- Activate via EBL inside the same Unity image (writes host-side entitlement) ---
+        - name: Activate Unity (EBL via container — host-mount)
+          if: steps.lic.outputs.use_ebl == 'true'
+          shell: bash
+          env:
+            UNITY_IMAGE:    ${{ env.UNITY_IMAGE }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 14 but found 12

(indentation)